frontend/hooks/useDeleteItem.ts

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/navigation'
+import { useCustomToast } from './useCustomToast'
+
+export const useDeleteItem = () => {
+  const router = useRouter()
+  const { showSuccessToast, showErrorToast } = useCustomToast()
+
+  const deleteItem = async ({ id, itemToDelete }) => {
```

Comment:
```suggestion
  const deleteItem = async ({ id, deleteTarget }: {
  id: string , deleteTarget: string) => {
```

みたいな方がいいかなと思いました。

itemToDelete
→抽象度が高い気がしました！

Reply:
該当箇所全て、`deleteTarget` へ変更しました！